### PR TITLE
[ENHANCEMENT] Added sorting by contact name to findUserByAccount function in UserRepository

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+trait OrderByTrait
+{
+    /**
+     * Function adds order-by to querybuilder based on $sortBy-data given.
+     *
+     * @param QueryBuilder $queryBuilder
+     * @param array $sortBy
+     */
+    private function addOrderBy(QueryBuilder $queryBuilder, $alias, $sortBy = [])
```
Comment (wachterjohannes) [inline src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php]: write a testcase for this trait!

File: src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Persistence\Repository\ORM;
+
+use Doctrine\ORM\QueryBuilder;
+
+trait OrderByTrait
```
Comment (wachterjohannes) [inline src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php]: phpdoc (:

File: src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Persistence\Repository\ORM;
```
Comment (wachterjohannes) [inline src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php]: @chirimoya mentioned we should create an own namespace `Sulu\Component\Persistence\Repository\ORM\Behavior` for that

Comment (turbo-ele) [inline src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php]: just talked to him. i changed his mind ;)

**Comment by wachterjohannes** [comment]
failing testcase

**Pull request merged**